### ISSUE TITLE
CMake Error 

### ISSUE THREAD

**Issue by makeclean** [openmc-dev/openmc#1783]
With the new build settings for submoduling or building our own dependencies, I now see a cmake failure

```
-- The C compiler identification is GNU 10.2.1
-- The CXX compiler identification is GNU 10.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring DAGMC 3.1.0
-- Did not find fmt, will use submodule instead
-- Found pugixml: /usr/lib64/cmake/pugixml
-- Found HDF5: /usr/lib64/libhdf5.so (found version "1.10.6") found components: C HL 
-- Found OpenMP_C: -fopenmp (found version "4.5") 
-- Found OpenMP_CXX: -fopenmp (found version "4.5") 
-- Found OpenMP: TRUE (found version "4.5")  
-- OpenMC C++ flags: -fopenmp;-O2
-- OpenMC Linker flags: -fopenmp
-- Found Git: /usr/bin/git (found version "2.29.2") 
-- Submodule update
-- Version: 6.1.2
-- Build type: 
-- CXX_STANDARD: 11
-- Performing Test has_std_11_flag
-- Performing Test has_std_11_flag - Success
-- Performing Test has_std_0x_flag
-- Performing Test has_std_0x_flag - Success
-- Performing Test SUPPORTS_VARIADIC_TEMPLATES
-- Performing Test SUPPORTS_VARIADIC_TEMPLATES - Success
-- Performing Test SUPPORTS_USER_DEFINED_LITERALS
-- Performing Test SUPPORTS_USER_DEFINED_LITERALS - Success
-- Performing Test FMT_HAS_VARIANT
-- Performing Test FMT_HAS_VARIANT - Success
-- Looking for strtod_l
-- Looking for strtod_l - not found
-- xtl v0.6.13
-- Building xtensor v0.21.3
-- Found xtl v0.6.13
-- Project 'gsl_lite', package 'gsl-lite' version: '0.36.1'
CMake Error at CMakeLists.txt:416 (set_target_properties):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "CXX_STANDARD" is not allowed.


CMake Error at CMakeLists.txt:416 (set_target_properties):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "CXX_EXTENSIONS" is not allowed.


-- Configuring incomplete, errors occurred!
See also "/home/adavis/opt/openmc-multiphysics/bld/CMakeFiles/CMakeOutput.log".
See also "/home/adavis/opt/openmc-multiphysics/bld/CMakeFiles/CMakeError.log".

```

I'm on cmake 3.18.4 if it helps.

**Comment by gridley** [comment]
Hm, I've only tested on 3.19. I don't see how that's related to any recent changes, because the `set_target_properties` line doesn't touch either pugixml or fmt. Are you on the develop branch?

**Comment by paulromano** [comment]
@makeclean Are you still seeing this behavior on the latest develop branch?

**Comment by gridley** [comment]
Closing due to inactivity, and pretty sure this was fixed in #1770 or #1794
